src/work-allocation-2/containers/all-work-task/all-work-task.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component } from '@angular/core';
-import { Location } from 'api/workAllocation2/interfaces/task';
-import { Caseworker } from 'api/workAllocation2/interfaces/task';
+import { Location } from 'api/workAllocation2/interfaces/common';
+import { Caseworker } from 'api/workAllocation2/interfaces/common';
```

Comment:
These two are coming from the `api/`, which they shouldn't.

Reply:
Resolved.